### DISCUSSION TITLE
Display just required wallet and Use projectId locally

### DISCUSSION THREAD

**Discussion by 99sudarshan** [WalletConnect/walletconnect-monorepo#3642] [Developer Support] [answered]
1. How to display just required wallets list in the **connect wallet popup**?
2.  How to verify project id to use locally **localhost:3000**?

**Comment by glitch-txs** [comment, marked as answer]
Hi you can add custom wallets following [this guide](https://docs.walletconnect.com/2.0/web3modal/react/wagmi/custom-wallets)
For question 2, the project ID comes pre-verified so it should work right away on localhost and production.
